src/main/java/com/epam/training/designpatterns/fastfoodrestaurant/Extra.java

```diff
@@ -0,0 +1,9 @@
+package com.epam.training.designpatterns.fastfoodrestaurant;
+
+public abstract class Extra implements Food {
```

Comment:
Helyesen használt decorator minta. Tankönyv szerint jogos az ősosztály használata, de az életben, nagyobb projektben lehet, hogy nem csinálnék ősosztályt az extráknak, mert túl erős csatolást vezet be, refactoringnak nem annyira barátja ilyenkor a kód. Tudom, ezzel megszegjük a DRY elvet...erről még lesz szó máshol.